lacework/provider.go

```diff
@@ -124,7 +124,6 @@ func Provider() *schema.Provider {
 			"lacework_resource_group_container":               resourceLaceworkResourceGroupContainer(),
 			"lacework_resource_group_gcp":                     resourceLaceworkResourceGroupGcp(),
 			"lacework_resource_group_machine":                 resourceLaceworkResourceGroupMachine(),
-			"lacework_team_member":                            resourceLaceworkTeamMember(),
```

Comment:
Do all your PR's need to go in any specific order of deployment? Or given no one is using it we should be fine?

Reply:
@nikhilesh-payyavuala See conversation here https://lacework.slack.com/archives/C04CYEQ3W3G/p1696444342248359

Because TeamMembers breaks the moment we migrated all customers to v2 we are now in a bit of predicament.

@afiune outliend the standard deprecation process for go cli/terraform here https://github.com/lacework/go-sdk/pull/1398#pullrequestreview-1658326828 but we can't really do the last step as it's broken for every customer already.